incskill_site/templates/quiz1.html

```diff
@@ -10,34 +10,33 @@
       <!-- creating a modal for when quiz ends -->
       <div class="modal-container" id="score-modal">
 
-          <div class="modal-content-container">
+          <div class="modal-content-container" id="last-modal">
 
-              <h1>Congratulations! Quiz Completed.</h1>
+              <h1 id="modal-title">Congratulations! Quiz Completed.</h1>
 
-              <div class="grade-details">
-                  <p>Attempts : 10</p>
+              <div id="grade-details">
                   <p>Wrong Answers : <span id="wrong-answers"></span></p>
                   <p>Right Answers : <span id="right-answers"></span></p>
                   <p>Grade : <span id="grade-percentage"></span>%</p>
                   <p ><span id="remarks"></span></p>
               </div>
 
-              <form method="post" class="modal-button-container">
+              <form method="post" class="modal-button-container" id="continue">
                 {% csrf_token %}
                 {{ form.as_p}}
-                  <button name="submit" id="submit" onclick="closeScoreModal()">Continue</button>
+                  <button name="continue" onclick="closeScoreModal()" id="continute-btn" style="background:rgb(43, 42, 42);">Continue</button>
```

Comment:
`continute` is misspelled also why do you have the inline style you should put that in the css not inline

Reply:
I was having trouble setting the background color of the button because I didn't realize continue was misspelled. Once I fixed that I was able to delete the inline style.